src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
```

Comment:
I contacted at irc with @mgol, after this interaction, i believe our discussion was productive.

Yet, from my point of view, if something is proposed to be changed, it should be tested before such proposal is submitted, i believe this is widely accepted practise, including our workflow too, in the same way, we ask all prs to have tests or good reason not to have it.